client/profile/user-reducer.ts

```diff
@@ -83,6 +83,10 @@ export default immerKeyedReducer(DEFAULT_STATE, {
     state.usernameToId.set(user.name, user.id)
   },
 
+  ['@chat/updateMessage'](state, action) {
+    updateUsers(state, [action.payload.user, ...action.payload.mentions])
```

Comment:
Prefer just calling `updateUsers` twice, once with each array. Less work to not allocate a new array and move all the references.

Apply throughout this reducer.